### ISSUE TITLE
Some quest issues since last update

### ISSUE THREAD

**Issue by Leander30** [AeroScripts/QuestieDev#710]
The following quests are showing as available again on the world map, even though I completed them. This happened since the last update. All of them are Horde quests.

- Barrens:
Quest: 
Flawed Power Stones (Showing up twice)

- Ashenvale:
Quests:
Warsong Runner Update
Warsong Scout Update
Warsong Outrider Update

- Tirisfal Glades:
Quests:
Candles of Beckoning
The Dormant Shade

**Comment by dotbanana** [comment]
I second this. It is not only this regions. In redridge mountains all quests show up as if i could take them but i have them already in questlog. This happened after my client crashed with some weit message that it could not do something with the graphics.

**Comment by CountessAnise** [comment]
Can confirm I'm having the same issue with "Flawed Power Stones".
It has been completed a few days ago (and still showing as completed on the "Character's Journey"). But since the last update it's showing up on the map (twice).

**Comment by AeroScripts** [comment]
Shift+Click is being added soon but, this seems to be a general bug with our GetQuestsCompleted logic. Not sure why only some are experiencing it though.

Can someone who is experiencing this try resetting questie to defaults, log out, disable all other addons and see if it behaves differently?


Sorry to anyone experiencing this, I know how frustrating it can be to be misdirected to an available quest

**Comment by AeroScripts** [comment]
Also, if anyone can post back and let me know your latency, that could be helpful. I have a hunch that latency could be the reason why only some are experiencing this (Which would be an easy fix)

**Comment by Leander30** [comment]
I only ever have around 30/33ms. Didn't try the reset yet though.

**Comment by AeroScripts** [comment]
Can you also try running these chat commands and tell me what they print?

/dump IsQuestFlaggedCompleted(926)

/run print(GetQuestsCompleted()[926] or "false")

and let me know what it prints to chat?

**Comment by CountessAnise** [comment]
These print:
`[1]=false`
And
`false`

As for the rest, will resetting also reset my "Character's Journey"? Because I really want to keep that =/

**Comment by AeroScripts** [comment]
yeah it will likely reset that. Don't worry about it though, those chat commands confirmed that a reset wouldn't help anyway. For whatever reason, both functions are saying you haven't completed flawed power stone. Those functions are the internal blizzard functions questie uses to check history, so no amount of resetting will fix it.

You're totally sure the quest isn't available again? Have you tried to accept it
If the game says you haven't finished it internally, I wouldn't be surprised at all if you're able to pick it up again.

**Comment by AeroScripts** [comment]
Actually, wowhead lists this quest as "Repeatable" also. Probably is repeatable.

**Comment by CountessAnise** [comment]
I went to the NPC to be sure. He offers no quest and I can't pickup the stone next to him to start the quest.
I completed it a few days ago.

But I understand it's not a Questie issue, as the internal commands like you said says it's not completed.
I will just ignore these for now and wait for the Shift+Click update.

The weird thing is that it was fine up until yesterday (or two days ago, not entirely sure).
I suppose Blizzard has changed something on their end.

Thank you for your research regarding this issue. And thank you for your contribution to the community!

**Comment by AeroScripts** [comment]
The repeatable version is started by the stone it's self actually (object) not an NPC at least according to wowhead. I'm almost at that level myself so ill do some testing. 

Candles of reckoning is another issue, I think we broke that one with changes to the available logic but it should be fixed soon. Next update will have shift+click hide also


We also changed the logic to properly respect GetQuestsCompleted() in the recent updates, it was causing issues otherwise. That may be why it wasn't happenig until just now. If GetQuestsCompleted really is messing up, I think we can figure out a way around it anyway.
